extension.js

```diff
@@ -219,6 +219,18 @@ function activate( context )
         } );
     }
 
+    function buildGlobs(includeGlobs, excludeGlobs) {
+        const globs = [];
+        for( const glob of includeGlobs ) {
+            globs.push(glob);
+        }
```

Comment:
We could just `concat` but I might try and play with this code to support the relative path thing that has been discussed (and has tripped me up too), so having it as an explicit loop just made that easier for tinkering.